### ISSUE TITLE
Fix ```get_vector_map``` for thermal plant and fuel consumption

### ISSUE THREAD

**Issue by carolmcs** [psrenergy/PSRClassesInterface.jl#87]
```julia
PSRI.get_vector_map(case_data,"PSRThermalPlant", "PSRFuelConsumption"; relation_type = PSRI.RELATION_1_TO_N) 
```
This code works for cases with at least one multi-fuel thermal plant (I won't put a working example for it, but you can ask me by message), but doesn't work when all thermal plants have only one fuel option:
```julia
import PSRClassesInterface
const PSRI = PSRClassesInterface

PATH_CASE_EXAMPLE_THER_PRICES = joinpath(pathof(PSRI) |> dirname |> dirname, "test", "data", "caso1")

data = PSRI.initialize_study(
    PSRI.OpenInterface(),
    data_path = PATH_CASE_EXAMPLE_THER_PRICES
)

ther2fuelcons = PSRI.get_vector_map(data,"PSRThermalPlant", "PSRFuelConsumption"; relation_type = PSRI.RELATION_1_TO_N)
```

**Comment by thiagonovaesb** [comment]
The get_vector_map function does not support 2-way relationships using just the JSON structure, but there is always an equivalent using get_reverse_vector_map. I opened the PR #89 with a fix proposal using the equivalent version.

**Comment by joaquimg** [comment]
closed by #89
